### DISCUSSION TITLE
gonarch always creates headcrabs & they grow up

### DISCUSSION THREAD

**Discussion by personguybutsmart** [VJ-HLR-Developers/Half-Life-Resurgence#81] [Ideas]
The gonarch could constantly create baby headcrabs, but when they see an enemy they make even more of them.
Also the babies could grow up after a certain amount of time (you could change the time near where you can select the baby headcrab limit)
Also would like to know if this is even possible lol

**Comment by KM-CM** [comment]
I'm not Vrej, but this IS possible (excluding any timer problems), but the crab would have to instantly snap to a grown up version (after some time, I mean). It's very hard to "interpolate models" from what I know, if not impossible.
